app/models/accredited_organization.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'accredited_representation/constants'
+
+class AccreditedOrganization < ApplicationRecord
+  # rubocop:disable Rails/HasAndBelongsToMany
+  has_and_belongs_to_many :accredited_individuals,
+                          class: 'AccreditedIndividual',
+                          join_table: 'accredited_individuals_accredited_organizations'
+  # rubocop:enable Rails/HasAndBelongsToMany
```

Comment:
Same with this one. https://www.rubydoc.info/gems/rubocop/0.58.1/RuboCop/Cop/Rails/HasAndBelongsToMany